ts/session/fetch_latest_release/index.ts

```diff
@@ -24,15 +24,23 @@ async function fetchReleaseFromFSAndUpdateMain(userEd25519SecretKey: Uint8Array)
     const diff = Date.now() - lastFetchedTimestamp;
     if (diff < skipIfLessThan) {
       window.log.info(
-        `[updater] fetched release from fs ${Math.floor(diff / DURATION.MINUTES)}minutes ago, skipping until that's at least ${Math.floor(skipIfLessThan / DURATION.MINUTES)}`
+        `[updater] fetched release from fs ${Math.floor(diff / DURATION.MINUTES)} minutes ago, skipping until that's at least ${Math.floor(skipIfLessThan / DURATION.MINUTES)}`
       );
       return;
     }
 
     const justFetched = await getLatestReleaseFromFileServer(userEd25519SecretKey);
-    window.log.info('[updater] fetched latest release from fileserver: ', justFetched);
+    if (!justFetched) {
+      window.log.info('[updater] no new release found on fileserver');
+      return;
+    }
+
+    const [releaseVersion, releaseChannel] = justFetched;
+    window.log.info(
+      `[updater] renderer process fetched from the ${releaseChannel} release channel on the fileserver: ${releaseVersion}`
+    );
 
-    if (isString(justFetched) && !isEmpty(justFetched)) {
+    if (isString(releaseVersion) && !isEmpty(releaseVersion)) {
```

Comment:
we should also check the type of releaseChannel I guess? before forwarding it blindinly as part of `ipcRenderer.send('set-release-from-file-server', justFetched);`